### PR TITLE
Add map to POI view (closes #40)

### LINKED ISSUE #40: Add map to POI view

### PR DESCRIPTION
**Pull request by wilsonpage**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,16 @@ With a [virtual or real device running](https://facebook.github.io/react-native/
...
 ```
 
+#### Using Android Studio (OSX)
+
+On OSX Android Studio must be launched from the command line in order to inherit `$PATH` so that a specific Gradle script inside Realm can run `npm --version`.
+
+Attempting to build using Android Studio that has been launched via the normal OSX icon will fail to build. We've included a shortcut script to help.
+
+```bash
+$ npm run launch-android-studio
```
Comment (gmarty) [inline README.md]: The `package.json` says it's `open-android-studio`.
Also we're using `yarn` in place of npm in the rest of the documentation, so we probably want to be consistent here.

File: js/scenes/Item.js [hunk 2]
```diff
@@ -51,26 +74,57 @@ class Item extends Component {
...
+      <View style={styles.content}>
+        <Animated.View
+          style={[styles.topLayer, { transform: [{ translateY: this.translateY }] }]}
+          {...this.panHandlers}>
+          <Header
+            title="Item"
+            navigator={this.navigator}
+            style={styles.header}/>
+          <Image
+            style={styles.image}
```
Comment (gmarty) [inline js/scenes/Item.js]: nit: let's move `style` to the last position in the attributes.

File: js/scenes/Item.js [hunk 3]
```diff
@@ -97,18 +192,37 @@ const styles = StyleSheet.create({
...
-    right: 0,
-    top: 0,
-    zIndex: 1,
     backgroundColor: 'rgba(0,0,0,0.4)',
   },
 
+  content: {
+    flex: 1,
+    position: 'relative',
+    backgroundColor: 'blue',
```
Comment (gmarty) [inline js/scenes/Item.js]: Why setting the background colour to blue? Is it for debugging purposes?

File: js/scenes/Item.js [hunk 4]
```diff
@@ -97,18 +192,37 @@ const styles = StyleSheet.create({
...
+    zIndex: 1,
+    flex: 1,
+    backgroundColor: '#222',
+    elevation: 4,
+  },
+
+  bottomLayer: {
+    ...StyleSheet.absoluteFillObject,
+    flex: 1,
+    backgroundColor: 'red',
```
Comment (gmarty) [inline js/scenes/Item.js]: Same here?

**Review by gmarty — APPROVED** [review]
I can't see anything wrong here but I'm totally new to both Animated and PanResponder so not sure if this review is entirely reliable.

I noticed that the transition from the list view is janky on the (si|e)mulators. Can you see that?

The position of the star and share component is out of place now. The marker looks buggy on iOS. Can you reproduce and fix those issues?

Finally, can you do a brown bag on PanResponder tomorrow? I went through the documentation but it'd be super valuable to share what you learned with us.

File: js/scenes/Item.js (hunk 3 above)
Comment (wilsonpage) [reply js/scenes/Item.js]: Ah yes, good catch. This was a last minute Friday night PR 😛

**Comment by wilsonpage** [comment]
Follow-up added https://github.com/mozilla-magnet/journey/issues/44

File: README.md (hunk 1 above)
Comment (wilsonpage) [reply README.md]: `yarn run` doesn't work for me :(

```bash
yarn run open-android-studio
yarn run v0.15.1
$ "open -a /Applications/Android\ Studio.app/ ." 
sh: open -a /Applications/Android\ Studio.app/ .: No such file or directory
```

**Pull request merged**